python/cairo-core/src/cairo_core/maths.cairo

```diff
@@ -554,50 +555,57 @@ func felt252_to_bits_rev{range_check_ptr, bitwise_ptr: BitwiseBuiltin*}(
         let range_check_ptr = range_check_ptr + 2;
     }
 
+    // Hint populates `bits_used` with the number of bits used to represent the value
+    // and writes the bits to `dst` in reversed order.
     let output = &dst[0];
-    // last non-zero bit
-    tempvar last_one: felt;
+    local bits_used: felt;
```

Comment:
why `local` instead of `tempvar`?

Reply:
it's a constant that we want to access at each iteration of the loop, without having to update it before jumping